hubedit/src/main.rs

```diff
@@ -93,6 +98,8 @@ pub enum Command {
         board: String,
         /// Git has for the hubris archive
         gitc: String,
+        /// Epoch for rollback protection
+        epoch: Option<u32>,
```

Comment:
Should this be optional here, or should it be a bare `u32` (with responsibility for deciding on a default value given to the caller)?